solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -243,12 +243,13 @@ pub mod audius_data {
         Ok(())
     }
 
-    pub fn write_track_social_action(
-        ctx: Context<TrackSocialAction>,
+    pub fn write_social_action(
```

Comment:
hmmm, not sure if social action is the best name because user social actions are not captured, but I can't think of a better name

Reply:
yeah makes sense i will update this